src/test/regress/output/multi_subquery.source

```diff
@@ -141,8 +144,8 @@ FROM
 		lineitem_subquery
 	GROUP BY
 		l_suppkey) AS order_counts;
-ERROR:  cannot push down this subquery
-DETAIL:  Group by list without partition column is currently unsupported
+ERROR:  cannot use real time executor with repartition jobs
+HINT:  Set citus.task_executor_type to "task-tracker".
```

Comment:
Probably not? This test became not relevant to this file since we're removing the flag.  We could even consider removing this test?